Composer/packages/client/src/pages/publish/publishStatusList.tsx

```diff
@@ -78,6 +73,7 @@ export const PublishStatusList: React.FC<IStatusListProps> = (props) => {
       isRowHeader: true,
       isResizable: true,
       data: 'string',
+      styles: { display: 'flex', alignItems: 'center' },
```

Comment:
Since u have reused the style 4 times i think u can create a variable and use it in the styles object